content/4_Gold/DSU.mdx

```diff
@@ -86,25 +86,35 @@ otherwise.
 #include <bits/stdc++.h>
 using namespace std;
 
-struct DSU {
-	vector<int> e;
-	DSU(int N) { e = vector<int>(N, -1); }
-
-	// get representive component (uses path compression)
-	int get(int x) { return e[x] < 0 ? x : e[x] = get(e[x]); }
+class DisjointSets {
```

Comment:
just curious if there's any particular reason for naming it this instead of DSU

Reply:
module names it this and i feel it makes the function a bit more obvious